### DISCUSSION TITLE
related missing API

### DISCUSSION THREAD

**Discussion by Nsingh0007** [sumitkolhe/jiosaavn-api#87] [General]
Have you removed the get Dashboard data API..?

**Comment by sumitkolhe** [comment]
Yes it was removed during the re-write. I'll add it back in upcoming release

**Reply by VamsiKrishna22042000** [reply]
I am not able to fetch songs based on artists, albums, playlists mainly english pop, english pop singers empty array is getting returned.

**Comment by Nsingh0007** [comment]
Some of the APIs are not working like (
https://saavn.dev/api/playlists?id=946682072).

Are there any Public API providers for music?
